wolox-books/src/app/screens/book/components/list/list.component.scss

```diff
@@ -25,3 +25,46 @@
   padding: 0 15px;
   width: 100%;
 }
+
+.add {
+  left: 163px;
+  position: absolute;
+  top: 263px;
+}
+
+.book-container {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.book-card {
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $soft-gray;
+  height: 300px;
+  margin-right: 36px;
+  margin-top: 36px;
+  padding: 28px 28px 0 28px;
```

Comment:
```suggestion
  padding: 28px 28px 0 0;
```

Reply:
(?) I need the padding to the left of the book card

Reply:
I think he wanted to say this: `padding: 28px 28px 0;`